packages/garbo/src/combat.ts

```diff
@@ -863,25 +863,42 @@ export class Macro extends StrictMacro {
   }
 }
 
-function customizeMacro<M extends StrictMacro>(macro: M) {
+type CustomizeMacroOptions = {
+  freeWanderer: (macro: StrictMacro) => Macro;
+  tentacle: (macro: StrictMacro) => Macro;
+  innateWanderer: (macro: StrictMacro) => Macro;
+};
+const DEFAULT_MACRO_OPTIONS = {
+  freeWanderer: () => Macro.kill(),
+  tentacle: () => Macro.basicCombat(),
```

Comment:
I don't remember why we use kill() vs basicCombat() for free wanderers, maybe they should both use basicCombat()?

Reply:
basicCombat includes stasising for hobo monkeys and involves deleveling; kill just kills. so actually it feels like if anything, basiccombat should be on the time prank or rubber spider. so eyah I agree, basic combat for both